### ISSUE TITLE
Voiding out in the Great Sky Island

### ISSUE THREAD

**Issue by Flax4** [MelonSpeedruns/TotkRandomizer#12]
https://github.com/MelonSpeedruns/TotkRandomizer/assets/50862920/c581712b-2f36-4210-b56b-e1dee968bc41

So not sure why this happens, is it a compatiblity issue? Does rando want a specific version of the game running? Or is it something else?

Running on:
Windows 11: 22631.3447
Ryujinx: 1.1.1272
Tears of the Kingdom version: 1.2.1
System Version: 18.0.0
Randomizer version: 1.0.0

**Comment by MelonSpeedruns** [comment]
I'm on 1.2.1 as well and this doesn't happen to me. The randomizer can be used with any version of the game.
Try doing a New Game? You shouldn't use the same save file between Randomizer versions.

**Comment by mrthethatone** [comment]
yeah im getting the exact same issue for some reason

**Comment by mrthethatone** [comment]
Right, so i found the fix. To get beyond that bridge MainChallenge needs to be at the "GetZonauMajic" Step.

**Comment by MelonSpeedruns** [comment]
Right, but I did set the default starting save to be one step after GetZonauMajic, so technically I thought that worked... hmm. Going to fix it then

**Comment by MelonSpeedruns** [comment]
The issue was found! Basically don't start the game without a save file, rather do "New Game" from the title screen.
I'll find a fix for it soon!

**Comment by Flax4** [comment]
> The issue was found! Basically don't start the game without a save file, rather do "New Game" from the title screen. I'll find a fix for it soon!

Can confirm this works for now, alright, time for me to finally goof around in TotK Rando!
